### DISCUSSION TITLE
First use and themes in publishconf.py

### DISCUSSION THREAD

**Discussion by livier** [getpelican/pelican#3429] [Q&A] [answered]
Hi, 
it is my first use. I began by "pelican-quickstart" and put a file "my-first-post". 
I put a "content.md" file. I installed a few themes "pelican-themes -i [themes from https://github.com/getpelican/pelican-themes] ... Then try with pelican (and make) commands. 

Nice `pelican -r -l "/my/way/content" -o "/my/way/output" -s "/my/way/pelicanconf.py"`
IT gives the web page localy as expected, with my text, and the theme. After setting "RELATIVE_URLS = True" in pelicanconf, I can navigage in the website. Ok

Now want to prepare the website I want to upload to my server and  try  ` pelican -r -l "/my/way/content" -o "/my/way/output" -s "/my/way/publishconf.py"`

I hope to check, at leat the first page at "http://127.0.0.1:8000/" as previous ... 
The theme as deseapear. Neverless I use original publishconf.py, just setup SITEURL  and the line "from pelicanconf import *" is present. I tryed to change RELATIVE_URLS = True in "publishconf.py" and then the theme appears again. 

Transfert the output directory to the server, it works. But  RELATIVE_URLS should be  = False on the server, no ? 

The  "publishconf.py" is now
```
# This file is only used if you use `make publish` or
# explicitly specify it as your config file.

import os
import sys

sys.path.append(os.curdir)
from pelicanconf import *

# If your site is available via HTTPS, make sure SITEURL begins with https://
SITEURL = "https://my-domain.tld/My-folder"


RELATIVE_URLS = True
#RELATIVE_URLS = False


FEED_ALL_ATOM = "feeds/all.atom.xml"
CATEGORY_FEED_ATOM = "feeds/{slug}.atom.xml"

# Delete the output directory, and all of its contents, before generating new files. This can be useful in preventing older, unnecessary files from persisting in your output. However, this is a destructive setting and should be handled with extreme care.
# DELETE_OUTPUT_DIRECTORY = True

# Following items are often useful when publishing

# DISQUS_SITENAME = ""
# GOOGLE_ANALYTICS = ""
```
I do not understand the meaning of the 3 firsts lines 
Is there an error i made somewhere, or what I do not understand ?

**Comment by dltj** [comment, marked as answer]
The SITEURL line — `SITEURL = "https://my-domain.tld/My-folder"` — implies that you want to serve your site on a sub-directory of the website root. Is that true?

It sounds like the path to the CSS file is not being calculated correctly. If you view the HTML source in the web browser, you'll see the path to the stylesheet near the top of the file:
```html
<link
      rel="stylesheet"
      type="text/css"
      href="-->>>URL here<<<--"
    />
```
The `URL here` value is probably wrong, and perhaps missing the sub-directory. Can you confirm that much?

**Reply by livier** [reply]
> The SITEURL line — `SITEURL = "https://my-domain.tld/My-folder"` — implies that you want to serve your site on a sub-directory of the website root. Is that true?
Yes

For the link to the stylesheet, seems you'r right too. 

I was testing the result on http://127.0.0.1:8000 before uploading this on the server. With RELATIVE_URLS = False, the link to the css file whent to the server, and finded nothing. After upload of the hole website, it finds what it needs there and the result seems good. 

All this can be confusing, even for my next steps, but  I undestand what can append now. Thanks for your help.
